plasmapy/particles/tests/test_atomic.py

```diff
@@ -626,10 +626,10 @@ def test_half_life():
 def test_half_life_unstable_isotopes():
     """Test that `half_life` returns `None` and raises an exception for
     all isotopes that do not yet have half-life data."""
-    for isotope in _isotopes.keys():
+    for isotope in _data_about_isotopes.keys():
```

Comment:
```suggestion
    for isotope in _data_about_isotopes:
```
I think the `.keys()` is unnecessary?  Need to check...